fleetctl/fleetctl.go

```diff
@@ -130,6 +133,9 @@ func init() {
 	globalFlagset.StringVar(&globalFlags.CertFile, "cert-file", "", "Location of TLS cert file used to secure communication with the fleet API or etcd")
 	globalFlagset.StringVar(&globalFlags.CAFile, "ca-file", "", "Location of TLS CA file used to secure communication with the fleet API or etcd")
 
+	globalFlagset.StringVar(&globalFlags.EtcdUsername, "etcd-username", "", "Username used to authenticate with etcd")
+	globalFlagset.StringVar(&globalFlags.EtcdPassword, "etcd-password", "", "Password used to authenticate with etcd")
+
```

Comment:
hmm, I'm not aware of any other variables that will work like that!
but if I'm not mixing things this code touches the old etcd API which was deprecated right ? and in that case do we need to expose these information ? it will be world readable and having a vulnerable node will just expose the whole cluster...

In fleetd it's fine, since it is retrieved from the configuration file. IMO the only thing to do if you are writing systemd drop-ins with such content is perhaps to make sure how you store that information.